@here/harp-mapview/lib/geometry/PhasedTileGeometryLoader.ts

```diff
@@ -161,10 +161,10 @@ export class PhasedTileGeometryLoader implements TileGeometryLoader {
         let didUpdate = false;
         toPhase = Math.min(toPhase, this.numberOfPhases);
         while (this.currentPhase < toPhase) {
-            didUpdate = this.update(enabledKinds, disabledKinds);
-            if (!didUpdate) {
+            if (!this.update(enabledKinds, disabledKinds)) {
                 break;
             }
+            didUpdate = true;
```

Comment:
Please update the api comment that this will now return true if only one phase update was successful, we may not have actually reached "toPhase".